LinusWKing/Teamwork-BackeEnd
# #29044397 Setup Testing Framework

#### What does this PR do?
Setup testing tools coupled with CI tools

#### Description of Task to be completed?

- Install Mocha and Chai packages

- Add basic Test file

- Integrate Travis CI as the Continous Integration Tool
#### How should this be manually tested?

- Clone this repository
- Run `npm install` to get all dependencies
- Run `npm run test` to run the test script
- Test should run and return a status

#### Any background context you want to provide?
No

#### What are the relevant project board stories?
[#29044397](https://github.com/LinusWKing/Teamwork-BackeEnd/projects/1#card-29044397)

#### Screenshots (if appropriate)
None

#### Questions:
None
